docs/docs/integrations/libraries/dbt/using-dbt-with-dagster-plus/serverless/existing-dbt-project.md

```diff
@@ -44,7 +44,7 @@ In this section, we'll demonstrate how to import an existing project to Dagster+
 The file structure of the repository will change the first time a project is deployed using Dagster+. For dbt projects, a few things will happen:
 
 - **A [`dagster_cloud.yaml` file](/deployment/code-locations/dagster-cloud-yaml) will be created.** This file defines the project as a Dagster+ code location.
-- **A few `.yml` files, used for CI/CD, will be created in `.github/workflows`.** [These files](/deployment/dagster-plus/ci-cd/ci-cd-file-reference), named `branch_deployments.yml` and `deploy.yml`, manage the deployments of the repository.
+- **A`.yml` workflow file, used for [CI/CD](/deployment/dagster-plus/deploying-code/ci-cd), will be created in `.github/workflows`.** This file, named `dagster-plus-deploy.yml`, manage the deployments of the repository.
```

Comment:
Something is off with this line. Not sure if it's supposed to be "A `.yml` workflow file,"